crew

```diff
@@ -334,28 +334,31 @@ def remove (pkgName)
     return
   end
 
-  Dir.chdir CREW_CONFIG_PATH do
-
-    #remove all files installed by the package
-    File.open("meta/#{pkgName}.filelist").each_line do |line|
-      begin
-        File.unlink line.chomp
-      rescue => exception #swallow exception
+  #if the package is marked as 'is_fake', the file/directorylists doesn't exist
```

Comment:
It is possible to have both an `is_fake` mark and file/directory list if it was regular package before but someone decide to convert it to empty place holder with `is_fake` mark.  I tried it first when I made a place holder last time.  Anyway, this is a comment to comment.  Not a real issue.